db/seeds.rb

```diff
@@ -5,3 +5,23 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+10.times do |n|
+    Event.create!(title:  "test#{n}",
+                 description: "This is description #{n}", 
+                 location: "location: #{n}",
+                 capacity: "12",
+                 registration_enabled: true,
+                 start_date: Time.zone.now, 
+                 end_date: 2.days.from_now)
+  end
+
+  13.times do |n|
+    Event.create!(title:  "Test#{n}",
+                 description: "This is description #{n}", 
+                 location: "location: #{n}",
+                 capacity: "12",
+                 registration_enabled: true,
+                 start_date: Time.zone.now - 5.days, 
+                 end_date: Time.zone.now - 2.days)
+  end
```

Comment:
I can remove this seeder if you guys want. Let me know!

Reply:
Please leave them, having seeds is a great idea and should've been done a long time ago! 👍

Reply:
Superb thanks!